types/ApiResponses.ts

```diff
@@ -390,7 +396,7 @@ export interface CancelAllPendingOrdersResponse extends ApiResponse {}
 
 export interface PublicResume {
   uuid: Uuid;
-  user: Uuid;
+  user: PublicProfile;
```

Comment:
nit: let's make this optional since not all `PublicResumes` need to return a `user` object (e.g. in the `getFullUserProfile()` case, resumes are queried by the user to populate `user.resumes`, so we don't need `user.resumes.user`.